# Genome assemblers

List of genome assembly programs for PacBio long-read datasets. Not in any specific order :D  
Some of the assemblers are suitable also for ONT reads.

1. Hifiasm

https://github.com/chhylp123/hifiasm  
Hifiasm is a fast haplotype-resolved de novo assembler for **PacBio HiFi reads**.  

2. IPA

https://github.com/PacificBiosciences/pbipa  
https://github.com/PacificBiosciences/pbbioconda/wiki/Improved-Phased-Assembler  
Improved Phased Assembler (IPA) is the official PacBio software for **HiFi** genome assembly.

3. CANU

https://github.com/marbl/canu  
https://canu.readthedocs.io/en/latest/tutorial.html  
Canu assembles reads from PacBio RS II or Oxford Nanopore MinION instruments into uniquely-assemblable contigs, unitigs. Both **raw and Hifi reads** supported.

4. FALCON

https://github.com/PacificBiosciences/pb-assembly  
FALCON and FALCON-Unzip are de novo genome assemblers for PacBio long reads, also known as Single-Molecule Real-Time (SMRT) sequences. Both **raw and Hifi reads** supported.

5. Wtdbg2

https://github.com/ruanjue/wtdbg2  
Wtdbg2 is a de novo sequence assembler for long noisy reads produced by PacBio or Oxford Nanopore Technologies (ONT). Both **raw and Hifi reads** supported.  

6. Flye

https://github.com/fenderglass/Flye  
Flye is a de novo assembler for single-molecule sequencing reads, such as those produced by PacBio and Oxford Nanopore Technologies. Both **raw and Hifi reads** supported.  

